app/index.js

```diff
@@ -215,6 +215,10 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       } else {
         console.log('ignoring auto updates during dev');
       }
+      // Disable accent pop-up
+      if (process.platform === 'darwin') {
+        systemPreferences.setUserDefault('ApplePressAndHoldEnabled', 'boolean', false);
```

Comment:
It is not the right place to do this. `setUserDefault`should be called only once and not at every window init.
Can you fix merge conflicts and move your block directly in `app.on('ready',...)` callback?